doc/conf.py

```diff
@@ -19,37 +20,41 @@
 source_suffix = ".rst"
 
 root_doc = "index"
-master_doc = "index"  # deprecated in 4.0
 
-project = "Qserv"
-
-# TODO
-copyright = ""
-
-author = "Rubin Observatory"
+# General information about the project.
+project = u'Qserv'
+copyright = u'2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)'
+author = u'LSST Data Management'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
-# TODO either set these manually here, or extract from e.g. env variables.
-version = "X.Y.Z"
-release = "X.Y.Z"
+
+# The short X.Y version.
+github_ref = os.getenv("GITHUB_REF", os.popen('git symbolic-ref HEAD').read().strip())
```

Comment:
two things:
one is that os.popen is going to run every time, which you don't really need. The other is that I think you need to close the handle returned by popen. So:
```
github_ref = os.getenv("GITHUB_REF")
if github_ref is None:
    with contextlib.closing(os.popen('git symbolic-ref HEAD')) as p:
        github_ref = p.read().strip()
```

Reply:
will do